pegasus/test/fixtures/fake_dashboard.rb

```diff
@@ -1,26 +1,42 @@
 require 'active_support'
 require 'active_record'
+require 'active_record/connection_adapters/sqlite3_adapter'
 require 'sequel'
 
-# Monkey-patch the SQLite connection adapter to ignore MySQL-specific schema creation statements.
-module ActiveRecord
-  module ConnectionAdapters
-    module SQLite3
-      class SchemaCreation < AbstractAdapter::SchemaCreation
-        private
+# Patch sqlite3 adapter to ignore MySQL-specific schema creation statements.
+module SchemaOptionFilter
+  def add_table_options!(create_sql, options)
+    if (options_sql = options[:options])
+      options_sql.gsub!(/ENGINE=\w+/, '')
+      options_sql.gsub!(/DEFAULT CHARSET=\w+/, '')
+      options_sql.gsub!(/COLLATE=\w+/, '')
+      create_sql << " #{options_sql}"
+    end
+  end
+end
+ActiveRecord::ConnectionAdapters::SQLite3::SchemaCreation.prepend SchemaOptionFilter
```

Comment:
I also updated the above patch to use `Module#prepend` instead of re-opening the class - I find this pattern makes the patch-application a bit more explicit.